tests/llmcompressor/transformers/obcq/test_consecutive_runs.py

```diff
@@ -48,10 +51,14 @@ def _test_consecutive_runs(
         stages = [stage.group for stage in session_recipe.stages]
         self.assertEqual(len(stages), 1)
         session.reset()
-
+        
+        recipe = resolve_recipe(recipe=self.first_recipe, model_path=self.output_first)
```

Comment:
Very nice, since we are no longer resolving the recipe within SparseAutoModel, we should probably put this in the main function since it's relevant for the finetuning pathway as well?

Reply:
Yeah thats what im thinking about - the code here runs after the first one shot is complete - so the simple solution would be to add this at the end -- will split it to another PR

Reply:
Okay